### PR TITLE
Prettier part2

### PR DESCRIPTION
**Pull request by bsmth**
Same again, slightly larger PR

### CODE REVIEW

File: create-media-stream-destination/index.html [hunk 1]
```diff
@@ -1,61 +1,61 @@
...
 
-       mediaRecorder.onstop = (evt) => {
-         // Make blob out of our blobs, and open it.
-         const blob = new Blob(chunks, { 'type' : 'audio/ogg; codecs=opus' });
-         document.querySelector("audio").src = URL.createObjectURL(blob);
-       };
-     }
+        mediaRecorder.onstop = (evt) => {
+          // Make blob out of our blobs, and open it.
+          const blob = new Blob(chunks, { type: "audio/ogg; codecs=opus" });
```
Comment (bsmth) [inline create-media-stream-destination/index.html:37]: Must double-check this is correct

File: spatialization/style.css [hunk 2]
```diff
@@ -1,475 +1,597 @@
...
-	justify-content: flex-end;
-	background: hsla(0, 0%, 100%, 0.5);
-	border: var(--border);
+  position: fixed;
+  bottom: 0px;
+  right: 0px;
+  display: flex;
+  justify-content: flex-end;
+  background: hsla(0, 0%, 100%, 0.5);
+  border: var(--border);
```
Comment (dipikabh) [inline spatialization/style.css:494]: Indention needed here?

File: spatialization/style.css [hunk 3]
```diff
@@ -1,475 +1,597 @@
...
+  display: inline-block;
+  margin: 0px;
+  padding: 3vmin 4vmin;
+  background-color: var(--green);
+  background-repeat: no-repeat;
+  background-position: center center;
+  background-size: 70% 70%;
+  border: var(--border);
+  border-radius: var(--borderRad);
+  cursor: pointer;
```
Comment (dipikabh) [inline spatialization/style.css:550]: indentation required here

**Review by dipikabh — CHANGES_REQUESTED** [review]

File: output-timestamp/index.html [hunk 4]
```diff
@@ -10,7 +10,7 @@
     <h1>Web Audio API examples: Output timestamp</h1>
     <button id="play">Play</button>
     <button id="stop" disabled="true">Stop</button>
-    <output><output/>
+    <output></output>
```
[the remark this answers is not in the document]
Comment (bsmth) [reply output-timestamp/index.html:13]: Script later on is injecting some content: https://github.com/mdn/webaudio-examples/blob/main/output-timestamp/index.html#L70

File: panner-node/style.css [hunk 5]
```diff
@@ -24,30 +25,31 @@ p {
...
 
-    position: absolute;
+  position: absolute;
 
-    box-shadow: inset 0px 0px 0px rgba(0,0,0,0.7);
-    transition: 0.1s all;
-    -webkit-transition: 0.1s all;
+  box-shadow: inset 0px 0px 0px rgba(0, 0, 0, 0.7);
+  transition: 0.1s all;
+  -webkit-transition: 0.1s all;
```
[the remark this answers is not in the document]
Comment (bsmth) [reply panner-node/style.css:43]: It looks correct to me:

```css
button {
  background-color: #0088cc;
  background-image: linear-gradient(to bottom, #0088cc 0%, #0055cc 100%);
  text-shadow: 1px 1px 1px black;
  text-align: center;
  color: white;
  border: none;
  width: 120px;
  font-size: 1.5rem;
  line-height: 1.5rem;
  padding: 0.5rem;

  position: absolute;

  box-shadow: inset 0px 0px 0px rgba(0, 0, 0, 0.7);
  transition: 0.1s all;
  -webkit-transition: 0.1s all;
}
```

File: create-media-stream-destination/index.html (hunk 1 above)
Comment (bsmth) [reply create-media-stream-destination/index.html:37]: Works as expected when running locally with `http-server`

File: panner-node/style.css [hunk 6]
```diff
@@ -24,30 +25,31 @@ p {
...
 }
 
-button:hover, button:focus {
+button:hover,
+button:focus {
```
[the remark this answers is not in the document]
Comment (bsmth) [reply panner-node/style.css:47]: Looks fine, tnx

File: spatialization/style.css (hunk 2 above)
Comment (bsmth) [reply spatialization/style.css:494]: Looks correct 👍🏻

File: spatialization/style.css (hunk 3 above)
Comment (bsmth) [reply spatialization/style.css:550]: Looks okay 👍🏻

**Review by dipikabh — APPROVED** [review]
Thank you 👍

**Pull request merged**